### PR TITLE
SB3 wrapper

### PR DESCRIPTION
**Pull request by takuseno**
@araffin
- Add `SB3Wrapper` class to provide compatible interfaces for SB3.
- Add `to_mdp_dataset` function to provide conversion from sb3's replay buffer to `MDPDataset`

### CODE REVIEW

File: d3rlpy/algos/base.py [hunk 1]
```diff
@@ -141,6 +141,7 @@ def fit_online(self,
                    update_start_step=0,
                    eval_env=None,
                    eval_epsilon=0.0,
+                   save_metrics=True,
```
Comment (araffin) [inline d3rlpy/algos/base.py:144]: is this related to SB3?

File: d3rlpy/wrappers/sb3.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+from gym.spaces import Discrete
+from d3rlpy.dataset import MDPDataset
+
+
+class SB3Wrapper:
+    """ A wrapper for Stable-Baseline3 (SB3) algorithms.
```
Comment (araffin) [inline d3rlpy/wrappers/sb3.py:6]:
```suggestion
    """ A wrapper for d3rlpy algorithms so they can be used with Stable-Baseline3 (SB3).
```

File: requirements.txt [hunk 3]
```diff
@@ -7,3 +7,4 @@ h5py==2.10.0
 gym==0.17.2
 kornia==0.4.0
 Cython==0.29.21
+stable-baselines3==0.10.0
```
Comment (araffin) [inline requirements.txt:10]:
```suggestion
stable-baselines3>=0.10.0
```
(as we are currently in beta, asking for the latest version is recommended)

**Review by araffin — COMMENTED** [review]
Minor remarks, otherwise LGTM ;)
Do you want me to create a PR to document that wrapper?

File: d3rlpy/algos/base.py (hunk 1 above)
Comment (takuseno) [reply d3rlpy/algos/base.py:144]: Actually, this is not related to SB3 wrapper. But, let me include this in this PR.

**Comment by takuseno** [comment]
@araffin Thank you for your checks! And, it'd be awesome if you create PRs for that! Thanks!!

File: d3rlpy/algos/base.py (hunk 1 above)
Comment (araffin) [reply d3rlpy/algos/base.py:144]: ok, no pb ;)

**Comment by araffin** [comment]
>@araffin Thank you for your checks! And, it'd be awesome if you create PRs for that! Thanks!!

I will see when I have some time ;) (I will try to do it this week but no promise)
